File: qc-application/src/main/java/com/eager/questioncloud/application/api/creator/service/CreatorService.java
package com.eager.questioncloud.application.api.creator.service;

import com.eager.questioncloud.core.domain.creator.dto.CreatorInformation;
import com.eager.questioncloud.core.domain.creator.infrastructure.CreatorRepository;
import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Service;

@Service
@RequiredArgsConstructor
public class CreatorService {
    private final CreatorRepository creatorRepository;

    public CreatorInformation getCreatorInformation(Long creatorId) {
        return creatorRepository.getCreatorInformation(creatorId);
    }
}

File: qc-application/src/main/java/com/eager/questioncloud/application/api/creator/service/CreatorService.kt
package com.eager.questioncloud.application.api.creator.service

import com.eager.questioncloud.application.api.creator.dto.CreatorInformation
import com.eager.questioncloud.application.api.creator.implement.CreatorReader
import org.springframework.stereotype.Service

@Service
class CreatorService(
    private val creatorReader: CreatorReader
) {
    fun getCreatorInformation(creatorId: Long): CreatorInformation {
        return creatorReader.getCreatorInformation(creatorId)
    }
}
